src/runaqa.ts

```diff
@@ -1,324 +1,388 @@
+        --delete-orphans --site  https://mirrors.kernel.org/sourceware/cygwin/
+        --local-package-dir "C:\\cygwin_packages"
+        --root "C:\\cygwin64"`)
+        await exec.exec(
+          `C:/cygwin64/bin/git config --system core.autocrlf false`
+        )
+        core.addPath(`C:\\cygwin64\\bin`)
+      }
+    } catch (error) {
+      if (error instanceof Error){
+        core.warning(error.message)
+      }
+      else{
+        core.warning('Unexpected error')
+      }
+    }
+    const antContribFile = await tc.downloadTool( 
+      `https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.zip/download`
+    )
+    await tc.extractZip(`${antContribFile}`, `${tempDirectory}`)
+    await io.cp(
+      `${tempDirectory}/ant-contrib/lib/ant-contrib.jar`,
+      `${process.env.ANT_HOME}\\lib`
+    )
+  } else if (process.platform === 'darwin') {
+    await exec.exec('brew install ant-contrib')
+    await exec.exec('sudo sysctl -w kern.sysv.shmall=655360')
+    await exec.exec('sudo sysctl -w kern.sysv.shmmax=125839605760')
+  } else {
+    if (fs.existsSync('/usr/bin/apt-get')) {
+      // Debian Based
+      await exec.exec('sudo apt-get update')
+      await exec.exec('sudo apt-get install ant-contrib -y')
+    } else if (fs.existsSync('/usr/bin/yum')) {
+      // RPM Based
+      await exec.exec('sudo yum update -y')
+      await exec.exec('sudo yum install p7zip -y')
+      const antContribFile = await tc.downloadTool(
+        `https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.zip/download`
+      )
+      await tc.extractZip(`${antContribFile}`, `${tempDirectory}`)
+      await io.cp(
+        `${tempDirectory}/ant-contrib/lib/ant-contrib.jar`,
+        `${process.env.ANT_HOME}\\lib`
+      )
+    } else if (fs.existsSync('/sbin/apk')) {
+      // Alpine Based
+      await exec.exec('apk update')
+      await exec.exec('apk add p7zip')
+      const antContribFile = await tc.downloadTool(
+        `https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.zip/download`
+      )
+      await tc.extractZip(`${antContribFile}`, `${tempDirectory}`)
+      await io.cp(
+        `${tempDirectory}/ant-contrib/lib/ant-contrib.jar`,
+        `${process.env.ANT_HOME}\\lib`
+      )
+    }
+    //environment
+    if ('RUNNER_USER' in process.env) {
+      process.env['LOGNAME'] = process.env['RUNNER_USER']
+    } else {
+      core.warning(
+        'RUNNER_USER is not the GitHub Actions environment variables shell script. Container is configured differently. Please check the updated lists of environment variables.'
+      )
+    }
+
+    if (fs.existsSync('/usr/bin/apt-get')) {
+      //disable apport

```

Comment:
why?

Reply:
If I change my Diff view to hide whitespace, I see a more accurate view of the code changes from @j-braley, I presume the editor being used has inserted additional or different whitespace, which causes it to look like the whole file changed.

Reply:
That would make sense. I knew it was something I did 😆

Reply:
notice how I blamed the IDE not you!  haha!